### DISCUSSION TITLE
Help : Token marked as a spam

### DISCUSSION THREAD

**Discussion by Hiroshicryptoff** [phantom/docs#327] [Help & Support] [answered]
### Summary

Our has been reported as spam. However, no bad practice on our part has been done. Moreover, it affects our community by not finding the token in their wallet or not being able to buy it on PHANTOM.

### Steps to Reproduce

Hello after the launch on Pump Fun, the token was considered spam on the Phantom wallet.

It was not during Pump Fun

With our community, we spent a lot of money on advertising on Dexscreener, Dextools Update...

Because of this, many were not able to buy the token directly on Phantom. And when they buy through Raydium, they encounter a difficulty to have it displayed with the other tokens in their wallet.

Legit Memecoin with a Community

Here is the solscan:

https://solscan.io/token/ibSKWQWZk9Gs4usUsrzjV8mHxWXGXFXhuxWHXsNpump

https://mumubull.fun

Social Networks

https://t.me/bullmumubullish
https://x.com/BullonsolBull

We have updated the token information on Dextools and Dexscreener

https://www.dextools.io/app/en/solana/pair-explorer/75i1jrvRaRuMqUU63Eyohp1z87xXkKrTA9ei4zmRm36Y?t=1733130461468

https://dexscreener.com/solana/75i1jrvrarumquu63eyohp1z87xxkkrta9ei4zmrm36y

Please help us to unmark the token as spam. There has been no spam from our community. And there is no reason for it to be considered that way.

Please help us urgently

### Phantom Version

24.26.0

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
Hello @Hiroshicryptoff,

It appears that this token no longer has the warning. Please let me know if you see differently.

**Reply by Brandoncoin** [reply]
thank you ! its now working perfectly

**Reply by rickyshashcoin** [reply]
Hello. I’m currently having this issue with RHC! The project is very well alive and thriving and I would love to get the yellow spam message removed from phantom! Please advise thanks! 
3BJH3qFDHQuamkv4E1ygCm7sZuJZuQUeNar6fjdNpump

**Comment by MomoAi-Lab** [comment]
Hey guys we have the same issue for our token.

9JbxQSKNukRA7cPZxCfhSNEcAP9iKRo3PSyYNbW4pump
